package.json

```diff
@@ -25,14 +25,14 @@
     "gray-matter": "^3.0.7",
     "hasha": "^3.0.0",
     "json-stable-stringify": "^1.0.1",
-    "pify": "^3.0.0",
-    "remark": "^8.0.0",
-    "remark-autolink-headings": "^5.0.0",
+    "rehype-autolink-headings": "^2.0.5",
+    "rehype-highlight": "^3.0.0",
+    "rehype-slug": "^2.0.3",
+    "rehype-stringify": "^6.0.0",
     "remark-gemoji-to-emoji": "^1.1.0",
-    "remark-highlight.js": "^5.0.0",
-    "remark-html": "^6.0.1",
-    "remark-inline-links": "^3.0.4",
```

Comment:
remark already supports references and definitions. `remark-inline-links` isn’t needed here, as it changes the markdown syntax tree to replace references and definitions with normal links. But because we’re going to HTML here everything gets changed to `<a>`s anyway